### PR TITLE
TargetFramework set to .net5.0

### LINKED ISSUE #189: Change framework to net5.0
### Follow dotnet 5 framework identifier evolution

I have Visual Studio 2019 16.6 the installed dotnet framework 5.0.100-preview.4.20258.7 wich have the framework identifier .net5 instead of netcoreapp5.0 .

So it cannot loads the projects within the solution,

### PR DESCRIPTION
**Pull request by alnikola**
TFM set to `.net5.0` in all project. Additionally, `rollForward` is enabled and set to `latestMinor`

**Note.** `dotnet new` does **not** support `.net5.0` TFM, yet.

Fixes #189

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "sdk": {
-    "version": "5.0.100-preview.6.20266.3"
+    "version": "5.0",
+    "rollForward": "latestMinor"
```
Comment (Tratcher) [inline global.json:4]: Why did you change this?

Comment (alnikola) [reply global.json:4]: Opening a solution directly without running our `startvs.cmd` only works if you have a proper SDK version in your shared `dotnet` folder (usually, inside Program Files). Previously, our `global.json` required the exact preview version like preview-6 and if you had only preview-5 it didn't work. Setting `rollForward` policy enables you to use any .NET 5 preview available on the machine.

Comment (Tratcher) [reply global.json:4]: startvs.cmd is required, there's no expectation that a developer have any version of the 5.0 SDK installed, let alone the right version.

Please revert.

Comment (alnikola) [reply global.json]: I see. Reverted.
